PWGCF/Flow/Tasks/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ o2physics_add_dpl_workflow(flow-pbpb-pikp
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::GFWCore
                     COMPONENT_NAME Analysis)
 
-o2physics_add_dpl_workflow(flow-gfw-omegaxi
+o2physics_add_dpl_workflow(flow-g-f-w-omega-xi
```

Comment:
I would suggest not changing the name of the executable
I would be easier and lesser impact to use `flowGfwOmegaXi` as the name of the source code file